app/services/diagnostics/move_inspector.rb

```diff
@@ -19,24 +20,26 @@ def generate
       @output << "time due:\t#{move.time_due}\n" if move.time_due.present?
       @output << "status:\t\t#{move.status}\n"
       @output << "cancel reason:\t#{move.cancellation_reason}\n" if move.cancellation_reason.present?
-      @output << "cancel comment:\t#{move.cancellation_reason_comment}\n" if move.cancellation_reason_comment.present?
-      @output << <<~ENDDETAILS
+      @output << "cancel comment:\t#{move.cancellation_reason_comment}\n" if include_person_details && move.cancellation_reason_comment.present?
+      @output << <<~ENDDETAILS1
```

Comment:
I think the heredoc naming doesn't actually get memoized beyond the scope of opening/closing the heredoc and I'm not sure that 1 and 2 add much information to other developers reading this.

Reply:
now renamed with new and improved names